src/test/ui/issues/issue-29516.stderr

```diff
@@ -1,4 +1,4 @@
-error: auto traits must not contain where bounds
+error: negative impls are not allowed for tuples
```

Comment:
do we disallow negative impls for all tuples? (I think that'd be ok, actually, though I think we could also permit them for tuples if they are only restricted based on arity, but I don't see the point.)

Reply:
I think allowing them for tuples will prevent us from ever adding variadic tuples, so it should be easier to just disallow them for now